examples/dual.html

```diff
@@ -3,8 +3,12 @@
 <head>
     <meta charset="utf-8">
     <title>Leaflet Sync Demo</title>
-    <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet/v1.0.3/leaflet.css" />
-    <script src="http://cdn.leafletjs.com/leaflet/v1.0.3/leaflet-src.js"></script>
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.3/dist/leaflet.css"
+      integrity="sha512-07I2e+7D8p6he1SIM+1twR5TIrhUQn9+I6yjqD53JQjFiMf8EtC93ty0/5vJTZGF8aAocvHYNEDJajGdNx1IsQ=="
```

Comment:
Do we really need the integrity attribute on these assets? I mean, it's just an example. I would say keeping that as clean as possible is something to persue